### ISSUE TITLE
Consider a catchier name

### ISSUE THREAD

**Issue by gkjohnson** [gkjohnson/three-gpu-pathtracer#158]
**Thoughts**

- ALICE (Advanced Light Intensity Calculation Engine)
- Daft Renderer
- Creola Renderer (NASA mathematician)
- Telescope Renderer
- Fiber Renderer

**Comment by erichlof** [comment]
I tend to like names that have some relation to the product/underlying software algo.
Here are some off the top of my head:
* Pathways renderer 
* (would have also suggested Pathfinder engine, but darn that Nissan, lol)
* Flashlight renderer (renders in a 'flash')
* Floodlight renderer (fills your scene with light)
* Spotlight renderer
* Searchlight renderer ('always follow the light')
* Limelight renderer (focuses attention on light)

I don't know, just some thoughts.  Coming up with project names is even harder than coming up with good variable names, lol.

**Comment by bhouston** [comment]
I think these 3 are not that desirable:
* "Light Ray Renderer" is too generic in my opinion.  It is just three descriptive generic names strung together.  So similar to:  https://lighttracer.org "Mental Ray Renderer", "V-Ray Renderer."
* "Flashlight" seems close to something XXX.
* "Spotlight renderer" it also generic and will be hard to search for.

The rest are fine.

My favourite is probably Limelight Renderer - https://en.wikipedia.org/wiki/Limelight. But there is a Rust WebGL library called: https://github.com/drifting-in-space/limelight but it isn't yet popular.  I think calling this "Limelight-Renderer" would be fine.

**Comment by erichlof** [comment]
I agree, the name has to be searchable among other projects. Lol about Flashlight - yeah I guess we don't want any x connotations, ha ha.
Limelight renderer has grown on me - it's pretty catchy and has something to do with light. 

On a side note, I had multiple rock bands in high school and naming them was always our most difficult rehearsals/meetings, ha!

**Comment by setpixel** [comment]
I like the current name! It's what it is exactly. No need to explain anything, easy to search for. Chef's kiss!
